spec/high_voltage/configuration_spec.rb

```diff
@@ -6,13 +6,11 @@
 
     before(:each) do
       HighVoltage.configure do |config|
-        ActiveSupport::Deprecation.silence do
```

Comment:
`ActiveSupport::Deprecation.silence` is deprecated. I was going to make this work with the new available API but then I realized we don't actually need this block anymore. It was added back when we deprecated caching. It's been a long time -- removing it doesn't even generate any warnings.